recipes/logr/0.2.0/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.14)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
```

Is the name of the target correct?
```suggestion
conan_basic_setup()

find_package(logr REQUIRED)

add_executable(example example.cpp)
target_link_libraries(example logr::logr)
```

---

If you mean `logr::logr`, then yes it is correct.

---

Yup, that's what I was asking. Thanks for checking!